### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by Flakebi**
- Add ninja to apt-get installs
- The build instructions on Ubuntu and RedHat are the same, so remove one
- Fix typo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,12 +78,12 @@ It is recommended to install at least 16GB RAM in your build system.
...
 ### Build System
 * CMake 3.15 or newer is required. [Download](https://cmake.org/download/) and install proper one if the cmake is older than 3.15.
-* Ninja is requred.
+* Ninja is required.
 
 ### Install Dev and Tools Packages
 #### Ubuntu
 ```
-sudo apt-get install build-essential curl g++-multilib gcc-multilib git pkg-config python3
+sudo apt-get install build-essential curl g++-multilib gcc-multilib git ninja-build pkg-config python3
```
Comment (qiaojbao) [inline README.md:86]: The build tools had been required above.
But this change is fine to me, I think we could add 'cmake' too.

File: README.md [hunk 2]
```diff
@@ -177,15 +177,6 @@ repo sync
...
-cmake -G Ninja -S xgl -B builds/Release64
-cmake --build builds/Release64
-
-cmake -G Ninja -S xgl -B builds/Release32 -DCMAKE_C_FLAGS=-m32 -DCMAKE_CXX_FLAGS=-m32
-cmake --build builds/Release32
-```
-#### RedHat
 ```
 cmake -G Ninja -S xgl -B builds/Release64
 cmake --build builds/Release64
```
Comment (qiaojbao) [inline README.md:182]: Could you help to remove the line 189: For RHEL7.X

**Review by qiaojbao — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Flakebi) [reply README.md]: Good point! I added it (and removed the RHEL7 line).

**Review by qiaojbao — APPROVED** [review]
LGTM

**Pull request merged**